website/main/content/blog/2025-03-16-architecture-to-scale.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Drawing inspiration from the Internet's architecture to scale consensus
+date: 2025-03-16
+draft: false
+description: Thinking about the foundational architecture to achieve Internet scale.
+tags: [ status-update, consensus ]
+authors: [ adlrocha ]
+---
+
+I want to kick-off my first weekly update in the project thanking Nazar for the warm welcome and the opportunity to work with him on this exciting project. I was really pumped to see other teams actively working on a similar problem to the one I started researching more than three years ago. For several reasons, I wasn't actively contributing to this problem any more, but this opportunity was the perfect excuse to get back to the game.
+
+> If you are curious about my previous work on the matter before joining Nazar, feel free to skim through [this paper](https://ieeexplore.ieee.org/abstract/document/9951359) to get the gist of it. 
```

Comment:
Is there a link to a freely accessible version of it?

Reply:
Good point, updated to include a public link.

Reply:
It is still the same link :thinking: